README.md

```diff
@@ -53,6 +53,7 @@ To the extent possible under law, the [SCATE] Project Team has waived all copyri
 
 If you reuse or make derivatives of this work, we request that you cite or otherwise give proper credit to this work.
 
+[Phenoscape Knowledgebase] http://beta.phenoscape.org/#/home
```

Comment:
Note that you _must_ put a colon after the square bracket. It is not correct Markdown syntax otherwise.